iroha_2_whitepaper.md

```diff
@@ -107,10 +99,21 @@ Additionally, the following two transaction types take as input (i.e., "wrap") o
 
 * Multisignature
 * Interledger (i.e., cross-chain)
+* Conditional Multisignature
+
+##### Conditional Multisignature:
 
-TODO:
+```
+[condition,
+   [signatory_set, ...],
+    ...
+]
+```
+
+where ```signatory_set``` is an m-of-n signatory set. If multiple ```signatory_set```s exist to a 
+
+As an example illustrating why conditional multisig is useful, consider a situation where a bank wants to allow either 2 tellers or 1 manager to sign off on any transfer transaction over $500 and under $1000. In this case, the condition will be: ```Condition.asset("usd@nbc").qty(500).comparison(">").qty(1000).comparison("<")``` and the ```signatory_set```s for the tellers and manager will be ```OR``` unified, so that either the m-of-n signatues from the tellers or the single signature from the manager will be acceptable for transaction signing.
```

Comment:
> Condition.asset("usd@nbc").qty(500).comparison(">").qty(1000).comparison("<")

1. should this condition only be attached to transaction?
2. When do we check this condition? On stateless or on stateful validation? or both?
3. what kind of conditions are enough to have for MVP?